Localization/StartMenuL10N.ini

```diff
@@ -2536,7 +2536,7 @@ Menu.RestartTip = Chiude tutti i programmi aperti e riavvia Windows.
 Menu.SleepTip = Mantiene la sessione in memoria e imposta la modalità basso consumo che consente di riprendere rapidamente il lavoro.
 Menu.HibernateTip = Salva la sessione e spegne il computer. Quando si riaccende il computer, la sessione verrà ripristinata.
 Menu.LogOffTip = Chiude i programmi e disconnette l'utente.
-Menu.DisconnectTip = Disconnette la sessione. È possibile riconnettersi a questa sessione al prossimo accesso.
+Menu.DisconnectTip = Disconnette dalla sessione. È possibile riconnettersi a questa sessione al prossimo accesso.
```

Comment:
Google Translate:
Disconnette la sessione. È possibile riconnettersi a questa sessione al prossimo accesso. - Disconnect the session. You can reconnect to this session the next time you log in.

Disconnette dalla sessione. È possibile riconnettersi a questa sessione al prossimo accesso. - Disconnect from the session. You can reconnect to this session the next time you log in.

Reply:
Both are fine. I tend to prefer the shorter forms (la) when used on tooltips.
But probably "dalla" is slightly more correct.